src/Models/Site.php

```diff
@@ -234,6 +234,46 @@ public function getName()
         return $this->get('name');
     }
 
+    /**
+     * Returns the site framework machine name.
+     *
+     * @return string
+     */
+    public function getFramework(): ?string
+    {
+        return $this->get('framework');
+    }
+
+    /**
+     * Returns TRUE if the site framework is Drupal 8.
+     *
+     * @return bool
+     */
+    public function isDrupal8Framework(): bool
+    {
+        return 'drupal8' === $this->getFramework();
+    }
+
+    /**
+     * Returns TRUE if the site framework is Drupal 7.
+     *
+     * @return bool
+     */
+    public function isDrupal7Framework(): bool
+    {
+        return 'drupal' === $this->getFramework();
+    }
+
+    /**
+     * Returns TRUE if the site framework is WordPress.
+     *
+     * @return bool
+     */
+    public function isWordpressFramework(): bool
+    {
+        return 'wordpress' === $this->getFramework();
```

Comment:
This should probably test to see if the framework begins with `wordpress`, to account for `wordpress-network`. Consider `isWordPressNetworkFramework()` and `isDrupalFramework()`.

Reply:
fixed